modules/op-changing-default-service-account.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-changing-default-service-account_{context}"]
+= Changing the default service account
+
+You can change the default service account by editing the `default-service-account` field in the `.spec.pipeline` and `.spec.trigger` specifications. The default service account name is `pipeline`.
+
+.Example: Default service account fields
```

Comment:
Same comment as previous re `.Example:`, I'm not sure about use of the semi-colon here. I think maybe just `.Example` is OK, and use callouts if necessary to describe in the code block what fields are being changed?